site-src/implementation-table.md

```diff
@@ -1,12 +1,19 @@
-This table is populated from the conformance reports uploaded by project implementations.
+The following tables are populated from the conformance reports uploaded by project implementations. They are separated into the extended features that each project supports listed in their reports.
+
```

Comment:
opted for having the projects along the x-axis and the features on the y-axis, since the long names made the scrolling tougher